Sources/BrowserServicesKit/ContentScopeScript/UserContentController.swift

```diff
@@ -113,12 +113,14 @@ final public class UserContentController: WKUserContentController {
     private let scriptMessageHandler = PermanentScriptMessageHandler()
 
     @MainActor
-    public init<Pub, Content>(assetsPublisher: Pub, privacyConfigurationManager: PrivacyConfigurationManaging)
+    public init<Pub, Content>(assetsPublisher: Pub, privacyConfigurationManager: PrivacyConfigurationManaging, earlyAccessScripts: [UserScript] = [])
```

Comment:
the naming here may be misleading since we‘re only installing the handlers and the scripts are ignored. A previous naming (`earlyAccessHandlers`) and some comment would do a better job IMO